src/main/java/com/parasoft/demoapp/config/rabbitmq/RabbitMQInventoryResponseQueueListener.java

```diff
@@ -32,6 +35,7 @@ public RabbitMQInventoryResponseQueueListener(MessageConverter rabbitMqMessageCo
 
     @Override
     public void onMessage(Message message) {
+        message.getMessageProperties().setHeader(DEFAULT_CLASSID_FIELD_NAME, InventoryOperationResultMessageDTO.class.getName());
```

Comment:
RabbitMQ listener in Virtualize can't send header key-value(__TypeId__=com.parasoft.demoapp.dto.InventoryOperationResultMessageDTO), I can't find a place to set header, so I oevrride the `__TypeId__` value to make the deserialization can work.